### DISCUSSION TITLE
Tailwind vscode plugin doesn't recognise css

### DISCUSSION THREAD

**Discussion by skorphil** [tailwindlabs/tailwindcss#15328] [Help]
Hi, i've installed TailwindCss vscode plugin to get rid of annoying vscode warnings in css when i use `@...` However now plugin seems to be conflicting with vscode. Initially .css files are treated as tailwind, but after i click them, they started to be treated as vanilla css and warnings again shown.

how to make css files be treated as tailwind?
I've tried to add:
```
"tailwindCSS.includeLanguages": {
    "html": "html",
    "javascript": "javascript",
    "typescript": "typescript",
    "vue": "vue",
    "css": "css",
    "scss": "scss",
  }
```
but it didnt help

Screen recording of a bug: https://youtu.be/U7L--MqcMas

**Comment by wongjn** [comment]
As per [the documentation](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss#files.associations):

> ### files.associations
>
> Use the `files.associations` setting to tell VS Code to always open `.css` files in Tailwind CSS mode:
>
> ```json
> "files.associations": {
>   "*.css": "tailwindcss"
> }
> ```
